.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

```suggestion
      interval: "monthly"
```
All other updates are run monthly here, so it would help to keep this in sync with that rhythm.